exercises/accumulate/README.md

```diff
@@ -66,7 +66,8 @@ The [exercism/rust](https://github.com/exercism/rust) repository on GitHub is th
 If you want to know more about Exercism, take a look at the [contribution guide](https://github.com/exercism/docs/blob/master/contributing-to-language-tracks/README.md).
 
 [help-page]: http://exercism.io/languages/rust
-[crates-and-modules]: http://doc.rust-lang.org/stable/book/crates-and-modules.html
+[modules]: https://doc.rust-lang.org/book/second-edition/ch07-00-modules.html
+[crates]: https://doc.rust-lang.org/book/second-edition/ch14-00-more-about-cargo.html
```

Comment:
differs from config/exercise-readme-insert.md. In particular, the name is `crates` here but `cargo` in config/exercise-readme-insert.md. I'm going to regenerate all READMEs to make them the same.